introduction.md

```diff
@@ -1,3 +1,7 @@
 Hi I'm Rowan
 
 My fav code school path is the Ruby one
+*Git
+*C#
+*Javascript
+*HTML
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?